src/code/hoc/animate-spring.js

```diff
@@ -16,16 +16,24 @@ import {Motion, spring} from 'react-motion';
     in its target state without further animation. This allows animations at rest to
     respond immediately to window resize events, for instance.
  */
-export default function animateSpring(stiffness=120, damping=25) {
+export default function animateSpring(stiffness=115, damping=18) {
```

Comment:
FYI, these defaults were chosen because they are (or were) the default values for the `react-motion` library, so the default behavior would be the same with or without the HOC.